dsa-solutions/lc-solutions/0000-0099/0075-sort-colors.md

```diff
@@ -0,0 +1,151 @@
+---
+id: sort-colors
+title: Sort Colors
+difficulty: Medium
+sidebar_label: 0075-sortcolors
+tags:
+  - Arrays
+  - Two Pointers
+  - Sorting
+---
+
+## Problem
+
+| Problem Statement | Solution Link | LeetCode Profile |
+| :---------------- | :------------ | :--------------- |
+| [Sort Colors](https://leetcode.com/problems/sort-colors/description/) | [Sort Colors Solution on LeetCode](https://leetcode.com/problems/sort-colors/solutions/) |  [Leetcode Profile](https://leetcode.com/u/debangi_29/) |
+
+## Problem Description
+
+Given an array nums with n objects colored red, white, or blue, sort them in-place so that objects of the same color are adjacent, with the colors in the order red, white, and blue.
+
+We will use the integers 0, 1, and 2 to represent the color red, white, and blue, respectively.
+
+You must solve this problem without using the library's sort function.
+ 
+
+### Examples
+
+### Example 1:
+
+**Input**: nums = [2,0,2,1,1,0]
+
+**Output**: [0,0,1,1,2,2]
+
+
+### Example 2:
+
+**Input**: nums = [2,0,1]
+
+**Output**: [0,1,2]
+
+ 
+
+### Constraints
+
+- $n = \text{nums.length}$
+- $1 \leq n \leq 300$
+- $\text{nums}[i] \in \{0, 1, 2\}$
+
+### Approach
+This problem is a variation of the popular Dutch National flag algorithm. 
+
+The steps will be the following:
+
+- First, we will run a loop that will continue until mid <= high.
```

Comment:
replace `mid <= high` to

```md
`mid <= high`
```

because `<` is introduce tags lisl (html/mdx, and also, jsx) so, not useing directly